.eslintcache

```diff
@@ -0,0 +1 @@
+[{"D:\\Professional\\Web dev\\my-app\\src\\index.js":"1","D:\\Professional\\Web dev\\my-app\\src\\reportWebVitals.js":"2","D:\\Professional\\Web dev\\my-app\\src\\App.js":"3","D:\\Professional\\Web dev\\my-app\\src\\Header.js":"4","D:\\Professional\\Web dev\\my-app\\src\\Content.js":"5","D:\\Professional\\Web dev\\my-app\\src\\Footer.js":"6","D:\\Professional\\Web dev\\my-app\\src\\UserCard.js":"7","D:\\Professional\\Web dev\\my-app\\src\\NewUser.js":"8"},{"size":510,"mtime":1609879743787,"results":"9","hashOfConfig":"10"},{"size":362,"mtime":499162500000,"results":"11","hashOfConfig":"10"},{"size":2477,"mtime":1610379237560,"results":"12","hashOfConfig":"10"},{"size":565,"mtime":1610030475740,"results":"13","hashOfConfig":"10"},{"size":1198,"mtime":1610384029667,"results":"14","hashOfConfig":"10"},{"size":101,"mtime":1609878638667,"results":"15","hashOfConfig":"10"},{"size":877,"mtime":1610409043043,"results":"16","hashOfConfig":"10"},{"size":2548,"mtime":1609882438531,"results":"17","hashOfConfig":"10"},{"filePath":"18","messages":"19","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"20"},"6uo8py",{"filePath":"21","messages":"22","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"20"},{"filePath":"23","messages":"24","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"25","messages":"26","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"20"},{"filePath":"27","messages":"28","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"29","messages":"30","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"20"},{"filePath":"31","messages":"32","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"33","messages":"34","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"20"},"D:\\Professional\\Web dev\\my-app\\src\\index.js",[],["35","36"],"D:\\Professional\\Web dev\\my-app\\src\\reportWebVitals.js",[],"D:\\Professional\\Web dev\\my-app\\src\\App.js",[],"D:\\Professional\\Web dev\\my-app\\src\\Header.js",[],"D:\\Professional\\Web dev\\my-app\\src\\Content.js",[],"D:\\Professional\\Web dev\\my-app\\src\\Footer.js",[],"D:\\Professional\\Web dev\\my-app\\src\\UserCard.js",[],"D:\\Professional\\Web dev\\my-app\\src\\NewUser.js",[],{"ruleId":"37","replacedBy":"38"},{"ruleId":"39","replacedBy":"40"},"no-native-reassign",["41"],"no-negated-in-lhs",["42"],"no-global-assign","no-unsafe-negation"]
```

Comment:
You should not include this file in the commit. Add .eslintcache to the git ignore and remove it from the commit.